tests/integration/test_api_literature.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
```

Comment:
sure if @jacquerie is ok with this. I believe he proposed it in order to have `test_api_authors` etc and keep the the files in order that way.